docs/operation/lerobot_guide.rst

```diff
@@ -0,0 +1,306 @@
+
+   - `--warmup-time-s`: Number of seconds for device warmup (default: 10s)
+   - `--episode-time-s`: Number of seconds per episode (default: 60s)
+   - `--reset-time-s`: Time for resetting after each episode (default: 60s)
+   - `--num-episodes`: Number of episodes to record (default: 50)
+
+   Example:
+
+   .. code-block:: bash
+
+      python lerobot/scripts/control_robot.py record \
+         --robot-path lerobot/configs/robot/aloha.yaml \
+         --fps 30 \
+         --root data \
+         --repo-id ${HF_USER}/aloha_test \
+         --tags tutorial \
+         --warmup-time-s 5 \
+         --episode-time-s 30 \
+         --reset-time-s 30 \
+         --num-episodes 2
+
+.. note::
+   To push your dataset to Hugging Face's Hub, log in with a write-access token:
+
+   .. code-block:: bash
+
+      huggingface-cli login --token ${HUGGINGFACE_TOKEN} --add-to-git-credential
+
+2. Set your Hugging Face username as a variable for ease:
+
+   .. code-block:: bash
+
+      HF_USER=$(huggingface-cli whoami | head -n 1)
+
+Visualizing Datasets
+====================
+
+To visualize all the episodes recorded in your dataset, run:
+
+.. code-block:: bash
+
+   python lerobot/scripts/visualize_dataset_html.py \
+      --root data \
+      --repo-id ${HF_USER}/aloha_test
+
+To visualize a single dataset episode from the Hugging Face Hub:
+
+.. code-block:: bash
+
+   python lerobot/scripts/visualize_dataset.py \
+      --repo-id TrossenRoboticsCommunity/aloha_static_block_pickup \
+      --episode-index 0
+
+To visualize a single dataset episode stored locally:
+
+.. code-block:: bash
+
+   DATA_DIR='./my_local_data_dir' python lerobot/scripts/visualize_dataset.py \
+      --repo-id TrossenRoboticsCommunity/aloha_static_block_pickup \
+      --episode-index 0
+
+Replay Recorded Episodes
+========================
+
+Replaying episodes allows you to test the repeatability of the robot's actions. To replay the first episode of your recorded dataset:
+
+.. code-block:: bash
+
+   python lerobot/scripts/control_robot.py replay \
+      --robot-path lerobot/configs/robot/aloha.yaml \
+      --fps 30 \
+      --root data \
+      --repo-id ${HF_USER}/aloha_test \
+      --episode 0
+
+.. tip::
+   Use different `--fps` values to adjust the frequency of the robot actions.
+
+Troubleshooting
+===============
+
+.. warning::
+   If you encounter issues, follow these troubleshooting steps:
+
+1. **OpenCV Installation Issues (Linux)**
+
+   If you encounter OpenCV installation issues, uninstall it via `pip` and reinstall using Conda:
+
+   .. code-block:: bash
+
+      pip uninstall opencv-python
+      conda install -c conda-forge opencv=4.10.0
+
+2. **FFmpeg Encoding Error (`unknown encoder libsvtav1`)**
+
+   Install FFmpeg with `libsvtav1` support via Conda-Forge or Homebrew:
+
+   .. code-block:: bash
+
+      conda install -c conda-forge ffmpeg
+
+   Or:
+
+   .. code-block:: bash
+
+      brew install ffmpeg
```

Comment:
We only support the use of our robot on Linux Ubuntu. Should we still include this?